index.html

```diff
@@ -14,13 +14,14 @@
         <a href=""><img src="img/logo.svg" alt="logo"></a>  
       </div>
       <div class="toggle-button" type="toogle">
-        <span class="menu-icon"><img src="img/Menu.png" alt="menu"></span>
+        <i class="fa fa-bars menu-icon"></i>
       </div>
       <ul class="menu">
-        <li><a href="#" title="Go to title">About</a></li>
-        <li><a href="#" title="Go to services">Services</a></li>
-        <li><a href="#" title="Go to contact">Contact</a></li>
+        <li class="menu-item"><a href="#about-myself" title="Go to title">About</a></li>
+        <li class="menu-item"><a href="#work" title="Go to services">Services</a></li>
```

Comment:
- [ ] kindly consider sticking to the design as much as possible by adding the right `text` as the one given in the design to stay consistence with the requirement below 👇 
> You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).
<img width="378" alt="Screen Shot 2023-06-28 at 06 16 58" src="https://github.com/HossainAraf/Portfolio-Mobile-First/assets/60146030/a4e8903a-9cfa-4f41-94e8-81f79295a4e2">